clang/lib/DPCT/MapNames.cpp

```diff
@@ -4122,6 +4122,19 @@ std::unordered_map<std::string, MacroMigrationRule> MapNames::MacroRuleMap{
     {"cudaStreamDefault",
      MacroMigrationRule("cudaStreamDefault_rule", RulePriority::Fallback,
                         "cudaStreamDefault", "0")},
+
+    {"CU_LAUNCH_PARAM_BUFFER_SIZE",
+     MacroMigrationRule("kernel_param_rule", RulePriority::Fallback,
+                        "CU_LAUNCH_PARAM_BUFFER_SIZE", "DPCT_KERNEL_PARAM_SIZE",
```

Comment:
Could u migrate those macros to literal values (like 0,1,2)?

Reply:
It's quite API specific things, it's not common. From SYCL view, we don't want to introduce this kind new macros in dpct:: namespace.

Reply:
Updated code to use literal values